2023-06 release notes (#147)
ni/install-systemlink-enterprise: release-notes/2023-06/readme.md

# SystemLink Enterprise release 2023-06 Release Notes

> The 2023-06 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.

Reviewer: This should probably just say '2023-06 release' and not mention 'bundle'.

## Upgrading from the release year-release-month to the release year-release-month

- 'assetservice >= 0.0.283'

>     - Description: New service was introduced in SystemLink chart. Any PVC that contains assetservice in its name needs to be removed prior to the upgrade. The following privileges are being used: asset:Query, asset:Create, asset:Update, asset:Delete.

Reviewer 2: ```suggestion
    - Description: This release introduced a new service in SystemLink chart. Before you upgrade, remove any PVC with assetservice in its name. The following privileges are being used: asset:Query, asset:Create, asset:Update, asset:Delete.
```
Is there a less passive way to mention the privileges? What is using them?

Author: Reformatted it to:

```
- `assetservice 0.0.283`
    - New in 2023-06 release
    - This service uses the following privileges: asset:Query, asset:Create, asset:Update, asset:Delete
    - Before you upgrade, remove any PersistentVolumeClaims (PVCs) with `assetservice` in its name
 ```

>     - Installation guide: Remove pvc that have assetservice in its name.

Reviewer 2: PVCs?

Author: Defined it as "PersistentVolumeClaims (PVCs)" - 👍
